Dockerfile.builder

```diff
@@ -28,6 +28,7 @@ RUN apt-get update && \
       git \
       libc6-dev \
       libelf-dev \
+      iproute2 \
```

Comment:
The builder image relies on the cilium-runtime image. The latter already have some of the tooling from iproute2.

What needs to be done instead is: changing https://github.com/cilium/packaging/blob/master/Dockerfile.iproute2 to copy ss tool into the scratch iproute2 image there. And once this is done the cilium-builder image needs to `COPY --from=quay.io/cilium/cilium-iproute2:xxxx-xx-xx /bin/ss /bin/` to its own image.

Reply:
But one more clarification, does the bugtool rely on Dockerfile.builder which is why we need it in here as a dependency?

Reply:
Thanks for the clarification. I was try to fix this.

Reply:
@soumynathan why does it need to be in the builder image at all?

Reply:
> @soumynathan why does it need to be in the builder image at all?

Sorry, my bad, I should not have added it here.

Reply:
> The builder image relies on the cilium-runtime image. The latter already have some of the tooling from iproute2.
> 
> What needs to be done instead is: changing https://github.com/cilium/packaging/blob/master/Dockerfile.iproute2 to copy ss tool into the scratch iproute2 image there. And once this is done the cilium-builder image needs to `COPY --from=quay.io/cilium/cilium-iproute2:xxxx-xx-xx /bin/ss /bin/` to its own image.

So we need to touch/modify the 'cilium/packaging' repo along with the 'cilium/bugtool'.

Reply:
> The builder image relies on the cilium-runtime image. The latter already have some of the tooling from iproute2.
> 
> What needs to be done instead is: changing https://github.com/cilium/packaging/blob/master/Dockerfile.iproute2 to copy ss tool into the scratch iproute2 image there. And once this is done the cilium-builder image needs to `COPY --from=quay.io/cilium/cilium-iproute2:xxxx-xx-xx /bin/ss /bin/` to its own image.

Ok, I have pushed in a PR to add the 'ss' to the cilium-iproute2 image. Once it merges, then we can edit the https://github.com/cilium/cilium/blob/a9af78470f432a4f5ebf8ecdef8a1c8b344f6586/contrib/packaging/docker/Dockerfile.runtime#L55 to include the 'ss'.

Reply:
Now since the PR 'https://github.com/cilium/packaging/pull/15' merged, do we need to wait for the latest 'cilium-iproute2: xxxx-xx-xx' build to happen. The latest I see is the 2020-05-20.

Reply:
The new container build would have occurred automatically here: 

https://github.com/cilium/image-tools/runs/712053055?check_suite_focus=true

There's been some changes to the way these images are generated/tagged recently so I'm not sure whether the date-based tags will show up or if we just need to adjust the main Cilium runtime image to consume these SHA tags from the repo directly.

@errordeveloper are the main Cilium dockerfiles consuming these images yet? If not, it may make sense for @soumynathan to put this work on the backburner until that work is done and then revisit this PR.